src/api/types.ts

```diff
@@ -18,8 +18,8 @@ export type APIError = {
  */
 export type BuildPayTransactionParams = {
   address: Address; // The address of the wallet paying
-  chainId: number; // The Chain ID of the payment Network (only Base is supported)
-  chargeId: string; // The ID of the Commerce Charge to be paid
+  chargeId?: string; // The ID of the Commerce Charge to be paid
+  productId?: string; // The ID of the product being paid for
```

Comment:
Curious if these should be required?

Reply:
we'll add validation on the component itself